app/controllers/SprintsController.php

```diff
@@ -19,11 +19,17 @@ public function show(Sprint $sprint)
 	public function showWithLiveData(Sprint $sprint)
 	{
 		$phabricator = App::make('phabricator');
-		$currentSprint = $sprint->project->currentSprint();
 		$taskList = new TaskList($phabricator->queryTasksByProject($sprint->phid));
-		$burndown = new BurndownChart($sprint, $taskList, $phabricator->getTaskTransactions($taskList->getClosedTaskIDs()));
 
-		return View::make('sprint.view', compact('sprint', 'currentSprint', 'taskList', 'burndown'));
+		return View::make(
+			'sprint.view',
+			[
+				$sprint,
+				$sprint->project->currentSprint(),
+				$taskList,
+				new BurndownChart($sprint, $taskList, $phabricator->getTaskTransactions($taskList->getClosedTaskIDs()))
+			]
+		);
 	}
```

Comment:
I think it's best to avoid `compact` due to it's high degree of magicness, and it being a rather unexpected way to go about passing along an array of local variables. Both these increase the chance of human error, and machine error (automated refactoring tools and static code analysers are likely to not handle this 100% correctly).